features/step_definitions/cli.rb

```diff
@@ -198,9 +198,15 @@
   @result = @bg_processes.last.result
 end
 
-Given /^I check status of last background process$/ do
+Given /^I check status of( and pop)? last background process$/ do |pop|
```

Comment:
Perhaps `I check status( and pop)? of last background process` ?

Reply:
@xingxingxia Forget it. Let's leave it as it is now.

Reply:
@xingxingxia the phrasing of this step is confusing.  Do you mean what does `pop` mean here?

Reply:
The `pop` is the `push` opposite, which means removing an entry from one stack-like object like the `@bg_processes` array.